core/consensus/authority/impl/authority_manager_impl.cpp

```diff
@@ -316,6 +318,9 @@ namespace kagome::authority {
                "Create authority set graph root with id {}, taken from runtime "
                "storage",
                root_->current_authorities->id);
+    } else {
+      SL_ERROR(log_, "Failed to initialize authority manager; Try running recovery mode");
+      return set_id_from_runtime_res.as_failure();
```

Comment:
Probably, in this case should recalculate state by reading of all digest starts genesis. What do you think about it?